contracts/ParsecBridge.sol

```diff
@@ -505,7 +505,9 @@ contract ParsecBridge {
 
     // if transfer, make sure signature correct
     if (txn.txType == TxLib.TxType.Transfer) {
-      address signer = ecrecover(TxLib.getSigHash(txData), txn.ins[_inputIndex].v, txn.ins[_inputIndex].r, txn.ins[_inputIndex].s);
+      bytes memory prefix = "\x19Ethereum Signed Message:\n32";
+      bytes32 sigHash = keccak256(abi.encodePacked(prefix, TxLib.getSigHash(txData)));
```

Comment:
same question here like in https://github.com/parsec-labs/parsec-lib/pull/23/files#r225201133, why not use `keccak(prefix + txDataWithoutSig)` instead of `keccak(prefix + keccak(txDataWithoutSig))`?